package.json

```diff
@@ -5,8 +5,9 @@
   "dependencies": {
     "@bugsnag/js": "^7.16.1",
     "@bugsnag/plugin-react": "^7.16.1",
-    "@cosmjs/proto-signing": "^0.27.1",
-    "@cosmjs/stargate": "^0.27.1",
+    "@cosmjs/proto-signing": "^0.28.1",
+    "@cosmjs/stargate": "^0.28.1",
```

Comment:
Updating `@cosmjs` is required, since it's only with `0.28.1` that it allows to specify a custom account parser.